docs/src/Datapoint.doc.js

```diff
@@ -0,0 +1,140 @@
+    defaultCode={`
+<Datapoint size="lg" infoText="Contextual information for this Datapoint" title="Datapoint title" value="1.23M" trend={{value: 30, accessibilityLabel: "Trending up"}} helperText="Datapoint helper text" />
+`}
+    pilot
+  />,
+);
+
+// Specifying Component gives auto warnings about missing props
+card(
+  <PropTable
+    Component={Datapoint}
+    props={[
+      {
+        name: 'title',
+        type: 'string',
+        description: 'The header for the component',
+        required: true,
+      },
+      {
+        name: 'value',
+        type: 'string',
+        description: 'The main datapoint value (e.g., 1.23M)',
+        required: true,
+      },
+      {
+        name: 'trend',
+        type: '{| accessibilityLabel: string, value: string |}',
+        description: `Object detailing the trend value (change in time - e.g., +30%), and accessibilityLabel to describe the trend's icon (e.g., "Trending up")`,
+      },
+      {
+        name: 'trendSignal',
+        type: `'good' | 'bad' | 'neutral' | 'auto'`,
+        defaultValue: 'auto',
+        description: `A visual indicator whether the trend is considered "good", "bad" or "neutral". By setting \`trendSignal\` to \`auto\`, a positive trend will be considered "good", a negative trend will be considered "bad" and a trend of zero will be considered "neutral"`,
+      },
+      {
+        name: 'infoText',
+        type: 'string',
+        description: 'Contextual information displayed in a tooltip to describe the Datapoint',
+      },
+      {
+        name: 'size',
+        type: `'md' | 'lg'`,
+        defaultValue: 'md',
+        description: 'Used to set the size of the datapoint',
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description={`Be sure to localize the \`title\`, \`infoText\` and \`trendAccessibilityLabel\` props. Note that localization can lengthen text by 20 to 30 percent.`}
+  />,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection columns={2} title="Size">
+      <MainSection.Card
+        cardSize="md"
+        title="size=md"
+        defaultCode={`
+        <Datapoint size="md" title="Datapoint title" value="1.23M" />
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        title="size=lg"
+        defaultCode={`
+        <Datapoint size="lg" title="Datapoint title" value="1.23M" />
+`}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`Use \`trend\` to display the delta in the value of a Datapoint over time. Make sure to provide an \`accessibilityLabel\` when the trend is above or below zero.`}
+      title="Trend"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Flex direction="column" gap={4}>
+  <Datapoint title="Datapoint title" value="1.23M" trend={{value: 30, accessibilityLabel: "Trending up"}} />
+  <Datapoint title="Datapoint title" value="1.23M" trend={{value: 0, accessibilityLabel: ""}} />
```

Comment:
It looks like `0%` might be too close the number display?
![Screen Shot 2021-04-29 at 12 15 21 PM](https://user-images.githubusercontent.com/12059539/116605741-a63da600-a8e4-11eb-9252-2b22060f0973.png)

Reply:
This follows the design spec, but open to feedback!